packages/aragon-cli/src/commands/dao_cmds/utils/getRepoTask.js

```diff
@@ -2,7 +2,7 @@ import { DEFAULT_IPFS_TIMEOUT } from '@aragon/toolkit/dist/helpers/constants'
 //
 const LATEST_VERSION = 'latest'
 
-module.exports = {
+export default {
```

Comment:
**Todo**: Replace 
```js
export default { funcA, funcB }
```
with
```js
export funcA
export funcB
```
**And** make sure other modules importing it are compatible with the new syntax